castanet/counter.py

```diff
@@ -30,7 +30,14 @@ def match_comment(cast_dict):
 
 
 def count_whileloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of while loops in a Python program.
+
+    Args:
+        cast_dict: adds the number of while loops to a dictionary
```

Comment:
This should read `cast_dict: A dictionary of files and the corresponding CAST's`.